runtime/queries/esdl/highlights.scm

```diff
@@ -0,0 +1,78 @@
+; Keywords
+[
+  "module"
+  "using"
+  "single"
+  "multi"
+  "link"
+  "property"
+  "constraint"
+  "tuple"
+  "annotation"
+  "abstract"
+  "scalar"
+  "type"
+  "required"
+  "optional"
+  "extension"
+  "function"
+] @keyword
+
+(modifier) @keyword
+(extending) @keyword
+
+(module name: (identifier) @namespace)
+(object_type) @type
+
+(comment) @comment
+
+; Properties
+(property) @property
+(link) @property
```

Comment:
this might also want to be `@variable.other.member` but if it's a link like a URL it might make sense to use `@markup.link.url`

Reply:
The link is essentially a property identifier so I updated to use `@variable.other.member`. Thanks for recommendation!